docs/root/api/starting_envoy.rst

```diff
@@ -259,6 +259,22 @@ Specify a closure to be called when Envoy's engine emits a log message.
     NSLog("Envoy log: \(msg)")
   }
 
+~~~~~~~~~~~~~~~~~~~
+``setEventTracker``
+~~~~~~~~~~~~~~~~~~~
+
+Specify a closure to be called when Envoy's engine emits an event.
+
+**Example**::
+
+  // Kotlin
+  // This interface is pending for Kotlin
```

Comment:
do you need help with this? Or is this for a separate PR?

Reply:
It's a separate PR - we are going to sync with Jingwei and/or Don on Android work. Hopefully, they will be able to help with Android changes but these iOS/common changes need to land first .